### ISSUE TITLE
Audit screenreader experience

### ISSUE THREAD

**Issue by molenick** [molenick/midilab#18]
Critical areas:
- Visually informative gui components that doesn't have accessibility labels
- Audit overall screen reader experience - is it cumbersome or fluid? We want a good default experience that strikes a good balance between usability and accuracy. Longer term, we may want configurable verbosity if it's too challenging to design a good audio experience.
- Is the default tab behavior intuitive? In a web context it would be (we haven't changed it) but I haven't looked closely at how egui handles tabbing.

**Comment by molenick** [comment]
https://github.com/molenick/midilab/pull/54 deals with most of what was needed.

I'm not sure if the screen reader experience is good but it should be minimally functional and serve as a point to refine from.

One thing we'll need to address is how status/error messages are handled.

This is the criteria we want to fulfill: https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

Maybe egui/accesskit already has it.
